src/app/components/job-list/job-list.component.html

```diff
@@ -1,17 +1,24 @@
-<ng-container *ngFor="let job of jobs; let i = index">
-  <div class="card end-0 mb-2" (click)="onJobClicked(job, i)"> 
-    <div class="row">
-      <div class="col-3 text-center pt-3 text-light" [ngStyle]=" {'background-color': job?.jobTypeColor}">
-        <p>JOB</p>
-        <p>{{ i+1 }}</p>
-      </div>
-      <div class="col-9">
-        <p>ID: {{ job?.accountNumber }}</p>
-        <h5>{{ job?.jobType }}</h5>
-        <p>Appt: {{ job?.appointmentWindow }}</p>
-        <span>{{ job?.streetAddress }}</span>
-        <p>{{ job?.cityAndState }}</p>
+<ng-container *ngIf="!jobsResponse">
+  <div class="d-flex justify-content-center mt-4">
+    <h3>Loading Jobs . . .</h3>
```

Comment:
Do you think we can use ngSwitch here? Also I don't think this will display when the user clicks "refresh"

Reply:
You're right, looking into this.

Reply:
Resolved. Added a new variable for monitoring the loading/response state, when used in conjunction with ngSwitch allows for displaying either job list, "jobs loading" or "job machine broke"